plugin/powerControl/powerControl.yaml

```
@@ -0,0 +1,43 @@
+  PowerOffBatchIntervalSeconds: 60    
+  NodeBMCMapping:                 
+    craned1: "10.129.227.172"
+    craned2: "10.129.227.189"
+  ExcludeNodes:                    
+    - "craned[2-10]"
+    - "craned3"
+
+SSH:
+  User: "YOUR_SSH_USER"
+  Password: "YOUR_SSH_PASSWORD"
+  
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)